src/test/java/com/exasol/adapter/dialects/IntegrationTestConstants.java

```diff
@@ -0,0 +1,19 @@
+package com.exasol.adapter.dialects;
+
+import java.nio.file.Path;
+
+public final class IntegrationTestConstants {
```

Comment:
I think the constants should go to the test because they are only used in one place now

Reply:
yes. To be fair, everything should go right now to the same test, and remove the abstract. I'm not sure what is the refactor you have in mind for this after is merged.

The idea of this PR was to just migrate the code and then refactor whatever is needed.

Maybe this should go in another PR.